schema/deploy/table_user.sql

```diff
@@ -5,7 +5,7 @@ begin;
 
 create table ggircs_portal.user
 (
-  id int not null,
+  id serial not null unique,
```

Comment:
I thought we were going with `not null generated always as identity`? This is currently not a primary key...and needs to be.